src/current/_includes/v23.1/ui/statement-details.md

```diff
@@ -62,6 +62,7 @@ The plan table shows the following details:
 Column | Description
 -----|----
 Plan Gist | A sequence of bytes representing the flattened tree of operators and operator-specific metadata of the statement plan.
+Used Indexes | The table indexes used by the plan. To see [table details]({{ link_prefix }}ui-databases-page.html#table-details), click on the table name. To see [index details]({{ link_prefix }}ui-databases-page.html#index-details), click on the index name.
```

Comment:
Should we link "table indexes" to https://www.cockroachlabs.com/docs/stable/create-index.html?

Reply:
Updated Statements page, in Explain Plans section, in Used Indexes column added indexes link: https://www.cockroachlabs.com/docs/stable/indexes.html